src/cookies.ts

```diff
@@ -15,56 +13,25 @@ const getShortDomain = (): string => {
 
 const getDomainAttribute = (): string => {
     const shortDomain = getShortDomain();
-    return shortDomain === 'localhost' ? '' : ` domain=${shortDomain};`;
+
+    return shortDomain === 'localhost' ? '' : shortDomain;
 };
 
 const addCookie = (name: string, value: string, daysToLive?: number): void => {
-    const expires = new Date();
-
-    if (!isValidCookieValue(name) || !isValidCookieValue(value)) {
-        return;
-    }
+    const options: {
+        domain: string;
+        expires?: number;
+    } = {
+        domain: getDomainAttribute(),
```

Comment:
What happens when we give it `domain: ''`? Does it act the same way as if no domain option has been given?

Reply:
The library doesn't add the `domain` attribute if it  is an empty string